automation_tools/__init__.py

```diff
@@ -418,6 +418,11 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
+def setup_scap_client():
+    """Task to setup puppet-foreman_scap_client."""
+    run('yum -y puppet-foreman_scap_client', warn_only=True)
```

Comment:
We needed to do that for ABRT because the package was not available on RHEL 6 repos. This is not needed and the warn_only will not let the installation fail.